sdk/communication/communication-identity/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 1.1.0-beta.1 (Unreleased)
 
+- `CommunicationIdentityClient` added a new method `exchangeAADtokenForACStoken`.
```

Comment:
..that provides the ability to exchange a teams token for a communication accessToken.